src/test/java/org/apache/ibatis/parsing/XPathParserTest.java

```diff
@@ -204,56 +204,140 @@ private void testEvalMethod(XPathParser parser) {
     String usersNodeToString = parser.evalNode("/users").toString();
     String userNodeToString = parser.evalNode("/users/user").toString();
     String carsNodeToString = parser.evalNode("/users/user/cars").toString();
 
     String usersNodeToStringExpect =
       "<users>\n" +
-      "    <user>\n" +
-      "        <id>100</id>\n" +
-      "        <name>Tom</name>\n" +
-      "        <age>30</age>\n" +
-      "        <cars>\n" +
-      "            <car index=\"1\">BMW</car>\n" +
-      "            <car index=\"2\">Audi</car>\n" +
-      "            <car index=\"3\">Benz</car>\n" +
-      "        </cars>\n" +
-      "    </user>\n" +
-      "</users>\n";
+        "\t<user>\n" +
+        "\t\t<id>\n" +
+        "\t\t\t100\n" +
+        "\t\t</id>\n" +
+        "\t\t<name>\n" +
+        "\t\t\tTom\n" +
+        "\t\t</name>\n" +
+        "\t\t<age>\n" +
+        "\t\t\t30\n" +
+        "\t\t</age>\n" +
+        "\t\t<cars>\n" +
+        "\t\t\t<car index=\"1\">\n" +
+        "\t\t\t\tBMW\n" +
+        "\t\t\t</car>\n" +
+        "\t\t\t<car index=\"2\">\n" +
+        "\t\t\t\tAudi\n" +
+        "\t\t\t</car>\n" +
+        "\t\t\t<car index=\"3\">\n" +
+        "\t\t\t\tBenz\n" +
+        "\t\t\t</car>\n" +
+        "\t\t</cars>\n" +
+        "\t</user>\n" +
+        "</users>\n";
 
     String userNodeToStringExpect =
       "<user>\n" +
-      "    <id>100</id>\n" +
-      "    <name>Tom</name>\n" +
-      "    <age>30</age>\n" +
-      "    <cars>\n" +
-      "        <car index=\"1\">BMW</car>\n" +
-      "        <car index=\"2\">Audi</car>\n" +
-      "        <car index=\"3\">Benz</car>\n" +
-      "    </cars>\n" +
-      "</user>\n";
-
-  String carsNodeToStringExpect =
+        "\t<id>\n" +
+        "\t\t100\n" +
+        "\t</id>\n" +
+        "\t<name>\n" +
+        "\t\tTom\n" +
+        "\t</name>\n" +
+        "\t<age>\n" +
+        "\t\t30\n" +
+        "\t</age>\n" +
+        "\t<cars>\n" +
+        "\t\t<car index=\"1\">\n" +
+        "\t\t\tBMW\n" +
+        "\t\t</car>\n" +
+        "\t\t<car index=\"2\">\n" +
+        "\t\t\tAudi\n" +
+        "\t\t</car>\n" +
+        "\t\t<car index=\"3\">\n" +
+        "\t\t\tBenz\n" +
+        "\t\t</car>\n" +
+        "\t</cars>\n" +
+        "</user>\n";
+
+    String carsNodeToStringExpect =
       "<cars>\n" +
-      "    <car index=\"1\">BMW</car>\n" +
-      "    <car index=\"2\">Audi</car>\n" +
-      "    <car index=\"3\">Benz</car>\n" +
-      "</cars>\n";
+        "\t<car index=\"1\">\n" +
+        "\t\tBMW\n" +
+        "\t</car>\n" +
+        "\t<car index=\"2\">\n" +
+        "\t\tAudi\n" +
+        "\t</car>\n" +
+        "\t<car index=\"3\">\n" +
+        "\t\tBenz\n" +
+        "\t</car>\n" +
+        "</cars>\n";
 
     assertEquals(usersNodeToStringExpect, usersNodeToString);
     assertEquals(userNodeToStringExpect, userNodeToString);
     assertEquals(carsNodeToStringExpect, carsNodeToString);
   }
 
+  @Test
+  public void testIssue2080() {
```

Comment:
IMO,The method can add comment if  you want to indicate the method fix which issue,So, To modify another method name?